.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: python
+python:
+  - "3.6"
+jobs:
```

Comment:
where is it specified that this triggers on merge to a particular branch? We don't want this to happen for all the branches?

Reply:
Line number 8, 22 makes sure the build is triggered only on adding a release tag.

Reply:
What happens if I put on release tag on `foo` branch sdk will be released to which branch? `foo` or `master` ?

Reply:
It would be released to master. There wont be any code changes to the SDK since we are building the SDK from the master branch, but the version would be updated to the version of the release tag on FOO. 
Also if the version already exists in master the build would fail because the same version cannot be pushed to PyPi again. 

I did not consider this possibility mainly because master is usually the default for tagging the release .